app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -10,34 +12,53 @@ def initialize(form526_submission)
     end
 
     def run
-      assess_data
+      use_vro = Flipper.enabled?(:rrd_call_vro_service)
+      use_vro ? assess_data_with_vro : assess_data
+
       unless @claim_context.sufficient_evidence
         return @form526_submission.save_metadata(offramp_reason: 'insufficient_data')
       end
 
       add_medical_stats
 
-      pdf = generate_pdf
+      pdf_body = (use_vro ? generate_pdf_body_with_vro : generate_pdf.render)
       @claim_context.add_metadata(pdf_created: true)
-      upload_pdf(pdf)
+      upload_pdf(pdf_body)
```

Comment:
Since the VRO endpoint can't return a Prawn PDF object, I shuffled this code a bit so that everyone is just passing around binary bytestrings.